### PR TITLE
added api endpoint flag

### PR DESCRIPTION
**Pull request by xh3b4sd**
This PR makes the GS API endpoint configurable. 

RFR @JosephSalisbury @marians

### CODE REVIEW

File: commands/root.go [hunk 1]
```diff
@@ -10,6 +10,7 @@ import (
 var RootCommand = &cobra.Command{Use: config.ProgramName}
 
 func init() {
+	RootCommand.PersistentFlags().StringVarP(&cmdAPIEndpoint, "api-endpoint", "", config.APIEndpoint, "The swagger base path, to access the API")
```
Comment (JosephSalisbury) [inline commands/root.go]: I don't see why swagger is important here - this is the URL base path, right?

File: config/config.go [hunk 2]
```diff
@@ -31,6 +31,9 @@ const (
 )
 
 var (
+	// APIEndpoint is the swagger base path, to access the API.
```
Comment (JosephSalisbury) [inline config/config.go]: Same here

**Comment by marians** [comment]
I agree with what @JosephSalisbury commented.

One additional question: where is the PR introducing `gsclientgen.NewDefaultApiWithBasePath()`?

File: commands/root.go (hunk 1 above)
Comment (xh3b4sd) [inline commands/root.go]: Yes.

**Comment by xh3b4sd** [comment]
@marian `gsclientgen.NewDefaultApiWithBasePath` is generated by swagger and already there.

**Pull request merged**